basic_module/distribution/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "distribution"
+version = "0.1.0"
+authors = ["CodeChain Team <hi@codechain.io>"]
+edition = "2018"
+
+[dependencies]
+primitives = { git = "https://github.com/CodeChain-io/rust-codechain-primitives.git", version = "0.4" }
+coordinator = {package = "coordinator", path = "../../coordinator" }
+ckey = { package = "codechain-key", path = "../../key" }
```

Comment:
Add a new line please.

Reply:
@somniumism I recommend you to install EditorConfig for VSCode plugin.
I can help you if you have any problem while installing it.